docs/modules/ROOT/pages/vshn-managed/postgresql/restore.adoc

```diff
@@ -49,20 +49,17 @@ spec:
       disk: "80Gi"
   writeConnectionSecretToRef:
     name: postgres-creds-restored
-  compositionRef:
-    name: vshnpostgresrestore.vshn.appcat.vshn.io # <3>
+
 ----
 <1> The name of the instance you want to restore from
 <2> The backup name you want to restore
-<3> You need to specify that this is not a normal instance but a restored one
-+
+
 [NOTE]
 ====
 The restore process is the same as ordering a new instance.
-The only difference is the backup information in `spec.parameters.restore` and `spec.compositionRef` objects.
```

Comment:
You don't need to remove the whole thing, only remove the mention of the compositonRef